app/views/layouts/angular.html.haml

```diff
@@ -19,6 +19,7 @@
         ga('create', '#{ENV["GOOGLE_ANALYTICS_ID"]}', 'auto');
 
   %body{'ng-keydown' => 'keyDown($event)'}
+    %flash
```

Comment:
Moved flash out to the html body, it's fixed now (allows it to show up over the modal overlay)